source/portable/NetworkInterface/MPS2_AN385/ether_lan9118/smsc9220_eth_drv.c

```diff
@@ -1223,7 +1223,7 @@ uint32_t smsc9220_peek_next_packet_size( const struct
     }
     else
     {
-        rx_status_from_peek = -1;
+        rx_status_from_peek = ( uint32_t ) -1;
```

Comment:
I think it's better to remove this `else` branch.

Reply:
Correct, just removing the else branch is correct.